terraform/state-backup.tf

```diff
@@ -0,0 +1,11 @@
+terraform {
+  backend "s3" {
+    bucket  = "radius-tracker"
```

Comment:
There's a `radius-tracker` bucket defined as a resource. I don't think it's a good idea to let terraform manage the same bucket it uses for storing configuration.